### ISSUE TITLE
importer for surfaces with inner rings creates invalid POLYHEDRALSURFACE

### ISSUE THREAD

**Issue by hugoledoux** [3dcitydb/importer-exporter#282]
With the importer v5.3.0, if I import this file (https://gist.github.com/hugoledoux/c982e49294d92390ab491f6751a0d228) (which is a simple cube with a genus=1, the top and bottom surfaces have inner rings, see below), then the POLYHEDRALSURFACE is not valid. While the geometry is valid and the cityjson file too (so not an issue here).

![SCR-20230511-gya](https://github.com/3dcitydb/importer-exporter/assets/1546518/d4ed761b-71e9-4198-ae7d-a31c4e15d423)

Each surface stored in `surface_geometry` is fine it seems, but the `solid_geometry` is not. I get that:

```
SRID=7415;POLYHEDRALSURFACE Z (((0.0 0.0 0.0, 0.0 1.0 0.0, 1.0 1.0 0.0, 1.0 0.0 0.0, 0.0 0.0 0.0)), ((0.2 0.2 0.0, 0.7000000000000001 0.2 0.0, 0.5 0.7000000000000001 0.0, 0.2 0.2 0.0)), ((0.0 0.0 1.0, 1.0 0.0 1.0, 1.0 1.0 1.0, 0.0 1.0 1.0, 0.0 0.0 1.0)), ((0.2 0.2 1.0, 0.5 0.7000000000000001 1.0, 0.7000000000000001 0.2 1.0, 0.2 0.2 1.0)), ((0.0 0.0 0.0, 1.0 0.0 0.0, 1.0 0.0 1.0, 0.0 0.0 1.0, 0.0 0.0 0.0)), ((1.0 0.0 0.0, 1.0 1.0 0.0, 1.0 1.0 1.0, 1.0 0.0 1.0, 1.0 0.0 0.0)), ((1.0 1.0 0.0, 0.0 1.0 0.0, 0.0 1.0 1.0, 1.0 1.0 1.0, 1.0 1.0 0.0)), ((0.0 1.0 0.0, 0.0 0.0 0.0, 0.0 0.0 1.0, 0.0 1.0 1.0, 0.0 1.0 0.0)), ((0.5 0.7000000000000001 0.0, 0.7000000000000001 0.2 0.0, 0.7000000000000001 0.2 1.0, 0.5 0.7000000000000001 1.0, 0.5 0.7000000000000001 0.0), (0.2 0.2 0.0, 0.5 0.7000000000000001 0.0, 0.5 0.7000000000000001 1.0, 0.2 0.2 1.0, 0.2 0.2 0.0), (0.7000000000000001 0.2 0.0, 0.2 0.2 0.0, 0.2 0.2 1.0, 0.7000000000000001 0.2 1.0, 0.7000000000000001 0.2 0.0)))
```

Notice that the 3 "internal surfaces" of the "hole" are stored in one surface, with one outer ring and 2 inner rings.

**Comment by yaozhihang** [comment]
thanks @hugoledoux. I can reproduce the problem. It seems like a bug in the code. We'll fix it asap
https://github.com/3dcitydb/importer-exporter/blob/38eb584a88d74aa9a97fabf1b361ab724dc92227/impexp-core/src/main/java/org/citydb/core/database/adapter/postgis/GeometryConverterAdapter.java#L558

**Comment by clausnagel** [comment]
Fixed with #284. The fix will be contained in the next release. [Build](https://github.com/3dcitydb/importer-exporter#building) the tool from source if you need a fixed version before the next release.
